storefront-x.px.config.js

```diff
@@ -14,6 +14,7 @@ export default {
     '@storefront-x/cypress',
     '@storefront-x/schema-org',
     '@storefront-x/nprogress',
+    '@storefront-x/partytown',
```

Comment:
For now remove partytown from demos (if you merge main, google analytics and google tag manager should disappear from demos too).